packages/bundle-source/test/endo-script-format.test.js

```diff
@@ -9,6 +9,7 @@ import bundleSource from '../src/index.js';
  * @param {string} entry
  * @param {Options} options
  */
+// @ts-expect-error 'Options' could be instantiated with a different subtype of constraint 'Partial<any>'.
 const generate = async (entry, options = {}) => {
```

Comment:
```suggestion
const generate = async (entry, options) => {
```

This should suppress the error, but then you'll need to supply an empty object somewhere below